src/App.jsx

```diff
@@ -14,45 +16,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [goods, setGoods] = useState(initialGoods);
+  const [buttonStates, setButtonStates] = useState({
```

Comment:
There is no need to use such a complex state here. Consider using 4 states - sort type (blank string, alphabet, length), order, visible, and initial goods. If changing sort type or order -> change state of visible todos occurs.